valhalla/baldr/graphtileheader.h

```diff
@@ -602,7 +613,7 @@ class GraphTileHeader {
 
   // TODO: in v4, don't store this its superfluous information, the graphid has all we need
   // Base lon, lat of the tile
-  std::pair<float, float> base_ll_ = {0, 0};
+  std::array<float, 2> base_ll_ = {0, 0};
```

Comment:
im worried about binary compatibility a little bit here? are we sure that in memory these things are equivalent?